keystore/keystore_test.go

```diff
@@ -4,13 +4,13 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/ChainSafe/gossamer/crypto"
+	"github.com/ChainSafe/gossamer/crypto/sr25519"
```

Comment:
Should we include ed and secp in keystore test?

Reply:
done